clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/services/SecurityGroupService.groovy

```diff
@@ -54,10 +56,13 @@ class SecurityGroupService {
    */
   Map<String, String> getSecurityGroupIds(Collection<String> securityGroupNames, String vpcId = null, boolean failIfNotAllResolved = true) {
     if (!securityGroupNames) { return [:] }
-    DescribeSecurityGroupsResult result = amazonEC2.describeSecurityGroups()
-    Map<String, String> securityGroups = result.securityGroups.findAll { securityGroupNames.contains(it.groupName) && it.vpcId == vpcId }.collectEntries {
-      [(it.groupName): it.groupId]
-    }
+    def request = new DescribeSecurityGroupsRequest()
+      .withFilters(new Filter("group-name", securityGroupNames.toList()))
```

Comment:
I take it you've ran this locally with multiple security group names ... not sure the behavior of those filters with multiple values.

Otherwise LGTM.